source/logHandler.py

```diff
@@ -290,7 +290,8 @@ def formatException(self, ex):
 
 	def formatTime(self, record: logging.LogRecord, datefmt: Optional[str] = None) -> str:
 		"""Custom implementation of `formatTime` which avoids `time.localtime`
-		since it causes a crash under some versions of Universal CRT ( #12160, Python issue 36792)
+		since it causes a crash under some versions of Universal CRT when Python locale
+		is set to a Unicode one ( #12160, Python issue 36792)

```

Comment:
Might as well fix this spacing while here

```suggestion
		is set to a Unicode one (#12160, Python issue 36792)
```